shared/config.ts

```diff
@@ -24,7 +24,7 @@ export const NETWORKS: NetworkInfo[] = [
     rpcHttp: 'https://rpc.mainnet.lukso.network',
     token: {
       symbol: 'LYX',
-      supply: 42_000_000,
+      supply: '42000000000000000000000000',
```

Comment:
Suggesting to use web3js for this:

```suggestion
      supply: web3.utils.toWei('42000000', "ether"),
```

https://web3js.readthedocs.io/en/v1.10.0/web3-utils.html#towei

<img width="728" alt="image" src="https://github.com/lukso-network/wallet.universalprofile.cloud/assets/31145285/cb3fc039-1756-4e2a-bf48-2f7154e95d80">

Reply:
Makes sense, I will add this separately since this one got merged.